src/controllers/authController.js

```diff
@@ -101,3 +101,42 @@ exports.resetPassword = async (req, res, next) => {
     next(err);
   }
 };
+
+exports.login = async (req, res) => {
+	try {
+		const prevUser = await User.findOne({ email: req.body.email});
+		console.log(prevUser)
+		if (prevUser) {
+			const passMatch= await bcrypt.compare(
+				req.body.password,
+				prevUser.password
+			);
+
+			if (passMatch && req.body.email === prevUser.email) {
+				const token = createToken(prevUser)
+
+				res.status(200).json({
+					status: 'success',
+					message: 'Login successful',
+					token,
+				});
+			} else {
+				res.status(401).json({
+					status: 'unauthorized',
+					message: 'authentication error',
```

Comment:
please specify what the authentication error is here, e.g email or password is incorrect